src/appleseed/renderer/kernel/rendering/oiioerrorhandler.h

```diff
@@ -51,6 +51,11 @@ class OIIOErrorHandler
   : public OIIO::ErrorHandler
 {
   public:
+    OIIOErrorHandler() : OIIO::ErrorHandler()
```

Comment:
No need to call the base class' constructor, it's done automatically.

Also, can you move the constructor definition to the .cpp file?

Reply:
Ok :)